wolfssl/test.h

```diff
@@ -4606,6 +4606,118 @@ static WC_INLINE int myRsaDec(WOLFSSL* ssl, byte* in, word32 inSz,
 
 #endif /* NO_RSA */
 
+static WC_INLINE int myGenMaster(WOLFSSL* ssl, void* ctx)
+{
+    int       ret;
+    PkCbInfo* cbInfo = (PkCbInfo*)ctx;
+    
+    (void)ssl;
+    (void)cbInfo;
+    
+    WOLFSSL_PKMSG("Gen Master");
+    /* fall through original routine */
```

Comment:
Slight wording change, addition of `to`:

`/* fall through to original routine */`

Applies to similar comments below also.